torch/linalg/__init__.py

```diff
@@ -140,6 +140,73 @@
     (tensor(3.7417), tensor(11.2250))
 """)
 
+cond = _add_docstr(_linalg.linalg_cond, r"""
+linalg.norm(input, p=None, *, out=None) -> Tensor
+
+Computes the condition number of a matrix :attr:`input`,
+or of each matrix in a batched :attr:`input`, using the matrix norm defined by :attr:`p`.
+The condition number is defined as the matrix norm of
+:attr:`input` times the matrix norm of the inverse of :attr:`input`.
+
+This function supports real-valued, and only on CPU, complex-valued input.
```

Comment:
"real-valued" -> "floating point", since I don't think this supports integer inputs?

Reply:
Even more precisely, "float and double"